### ISSUE TITLE
Bug: o command open always the currently selected file from the library

### ISSUE THREAD

**Issue by ad-si** [2mol/pboy#41]
No matter if I'm currently focused on the library or the inbox, it always opens the currently selected file from the library.

I'm using the latest version on macOS:

```
$ brew info 2mol/tools/pboy
2mol/tools/pboy: stable 1.3
Small .pdf management utility
https://github.com/2mol/pboy
/usr/local/Cellar/pboy/1.3 (3 files, 5.5MB) *
  Built from source on 2020-03-31 at 15:53:29
```

**Comment by 2mol** [comment]
Oh hey, good catch! Fixed in the 1.4 dev branch: https://github.com/2mol/pboy/commit/b34336dc721f5a61a3a6594619e9945470d1c0e3

Will create a release once I collect a bunch more fixes and improvements. Unless you want it fast?

**Comment by ad-si** [comment]
👍 
No worries, my usage isn't mission critical. I just use it every now and then to sort through some downloaded PDFs.

**Comment by 2mol** [comment]
fixed and released in 1.4 with #45
